client/src/ECP.ts

```diff
@@ -51,11 +51,16 @@ export class ECP {
 		this.device.setConfig(config);
 	}
 
-	public getConfig() {
+	/** Provides a way to get the whole config not just this classes' Config */
```

Comment:
What about this message instead. I don't think we need to specify "not just this class's config", it would be better to simply include _what_ this function returns rather than what it doesn't. 
```suggestion
	/** 
	 * Get the full RTA config
	 */
```